e2e-tests/graphql-spring-examples/target/classes/petshop.graphqls

```diff
@@ -1,10 +1,13 @@
 type Query {
```

Comment:
comment related to the `target/classes/META-INF/evomaster-e2e-tests-graphql-spring-examples.kotlin_module ` above.
files under `target` should never be added to Git. add such folder on `.gitignore` file